pkg/kubelet/apis/config/types.go

```diff
@@ -108,6 +108,9 @@ type KubeletConfiguration struct {
 	// readOnlyPort is the read-only port for the Kubelet to serve on with
 	// no authentication/authorization (set to 0 to disable)
 	ReadOnlyPort int32
+	// readOnlyBindAddress is the address for the Kubelet to serve on with
+	// no authentication/authorization (set to 0.0.0.0 to serve on all interfaces)
+	ReadOnlyBindAddress string
```

Comment:
I expected to see validation if a non-empty value was specified in ValidateKubeletConfiguration

Reply:
ack. adding.